tests/test_marc.py

```diff
@@ -148,6 +148,33 @@ def test_marc_record_all_fields_transform_correctly():
             ),
         ],
         edition="9th ed. / Nick Ray ... [et al.]. Unabridged.",
+        identifiers=[
+            timdex.Identifier(value="2005022317", kind="LCCN"),
+            timdex.Identifier(value="9781250185969. hardcover", kind="ISBN"),
+            timdex.Identifier(
+                value="0878426914. paperback. alkaline paper", kind="ISBN"
+            ),
+            timdex.Identifier(value="0033-0736", kind="ISSN"),
+            timdex.Identifier(value="0095-9014", kind="ISSN"),
+            timdex.Identifier(
+                value="10.1596/978-0-8213-7468-9. doi", kind="Other Identifier"
+            ),
+            timdex.Identifier(value="1234567890. score. sewn", kind="Other Identifier"),
+            timdex.Identifier(value="(OCoLC)1312285564", kind="OCLC Number"),
+            timdex.Identifier(value="(OCoLC)on1312285564", kind="OCLC Number"),
```

Comment:
This repetition of OCLC identifiers is very common so we may want to consider a dedupe block

Reply:
Yeah, maybe make a ticket for that or add it to https://mitlibraries.atlassian.net/browse/TIMX-122?